.github/workflows/build-cache.yml

```diff
@@ -16,6 +16,11 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Save date of cache build
```

Comment:
We need to save it before we do the actual build so that if we later on set the mtime to that timestamp, the build artefacts will be slightly newer.